apps/ledger-live-desktop/src/renderer/components/Web3AppWebview/WalletAPIWebview.tsx

```diff
@@ -291,6 +294,9 @@ export const WalletAPIWebview = forwardRef<WebviewAPI, WebviewProps>(
 
     return (
       <>
+        {!webviewState.loading && webviewState.isAppUnavailable && (
+          <NetworkErrorScreen refresh={handleRefresh} />
+        )}
```

Comment:
[Question]: Are we still rendering the webview component in this case? If so do we want to do that or should we only render the webview if there is no network error?

Reply:
Thanks 👍